### DISCUSSION TITLE
Dark mode glitches in MV3

### DISCUSSION THREAD

**Discussion by krystian3w** [uBlockOrigin/uBlock-issues#2257] [General]
![obraz](https://user-images.githubusercontent.com/35370833/189507648-b2d5cb0a-9601-4024-9ee6-7ca3607686ab.png) |
--- |

**Comment by ghost** [comment]
Reproducible?

**Reply by krystian3w** [reply]
Dark theme in UnGoogled Chromium 105.0.5195.102 - maybe Google code made detect dark theme ~~and community cut too much of code~~ (I reporduced in 105 Edge, MS may not cut API to detect dark mode).

> ![obraz](https://user-images.githubusercontent.com/35370833/189529212-cd0ae20e-ff65-41f3-a4a7-ffed07ceaf9b.png) |
> --- |
